go/arrow/internal/arrjson/arrjson.go

```diff
@@ -475,6 +477,39 @@ func typeFromJSON(typ json.RawMessage, children []FieldWrapper) (arrowType arrow
 		case "DENSE":
 			arrowType = arrow.DenseUnionOf(fieldsFromJSON(children), t.TypeIDs)
 		}
+	case "runendencoded":
+		if len(children) != 2 {
+			err = fmt.Errorf("%w: REE array must have exactly 2 fields, but got %d",
```

Comment:
Antoine has commented on my PR suggesting that I change REE to "run-end encoded". I think that gives people something better to Google when they get these error messages without any extra context.

Reply:
fair point, i'll update the error messages here.